### PR TITLE
add new querypane header

### LINKED ISSUE #3624: Redesign Query pane layout
Update the Query pane layout to be full width

### LINKED ISSUE #3625: Update the Query pane header
Update the query pane header

### PR DESCRIPTION
**Pull request by ram-primathon**
## Description
Add new querypane header

Fixes #3624
Fixes #3625

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app/client/src/pages/Editor/routes.tsx [hunk 1]
```diff
@@ -202,7 +202,11 @@ const PaneDrawer = (props: PaneDrawerProps) => {
       dispatch(closeAllModals());
     }
   }, [dispatch, props.isVisible, selectWidget, showPropertyPane, focusWidget]);
-  return <DrawerWrapper {...props}>{props.children}</DrawerWrapper>;
+  return (
+    <DrawerWrapper {...props} isAPIPath={true}>
```
Comment (hetunandu) [inline app/client/src/pages/Editor/routes.tsx:206]: Why is this new prop needed?

File: app/client/src/pages/Editor/QueryEditor/Form.tsx [hunk 2]
```diff
@@ -406,87 +287,47 @@ const QueryEditorForm: React.FC<Props> = (props: Props) => {
...
-                onClick={onRunClick}
-              />
-            </OnboardingIndicator>
-          )}
-        </ActionsWrapper>
-      </StyledFormRow>
+                ) : null}
+              </HeaderIconHolder>
+              <div>
+                <ActionNameEditor page="API_PANE" />
```
Comment (hetunandu) [inline app/client/src/pages/Editor/QueryEditor/Form.tsx:318]: Can we remove this page prop now? I don't think it is needed anymore

**Review by hetunandu — CHANGES_REQUESTED** [review]

File: app/client/src/pages/Editor/routes.tsx (hunk 1 above)
Comment (ram-primathon) [reply app/client/src/pages/Editor/routes.tsx:206]: this was to make container with 100%. removed

File: app/client/src/pages/Editor/QueryEditor/Form.tsx (hunk 2 above)
Comment (ram-primathon) [reply app/client/src/pages/Editor/QueryEditor/Form.tsx:326]: it is getting used in RapidApiEditorForm

**Review by hetunandu — APPROVED** [review]

**Pull request merged**